### PR TITLE
Add ability to turn off importance sampling, create summary pages

### PR DESCRIPTION
**Pull request by stephengreen**
This makes two additions to dingo_pipe:

* `importance-sample` option (default True). By setting this to False, we can turn off importance sampling.
* `create-summary`. This is taken over from bilby_pipe. It calls PESummary at the end of the run.

### CODE REVIEW

File: dingo/pipe/dag_creator.py [hunk 1]
```diff
@@ -78,63 +79,67 @@ def generate_dag(inputs):
...
+                #       (Automatically calculates evidence.)
+                merge_node = MergeNode(
+                    inputs=inputs,
+                    parallel_node_list=parallel_node_list,
+                    dag=dag,
+                )
+                merged_importance_sampling_node_list.append(merge_node)
+
+    else:
+        merged_importance_sampling_node_list = sampling_node_list
```
Comment (mpuerrer) [inline dingo/pipe/dag_creator.py:142]: Consider renaming `merged_importance_sampling_node_list` to `merged_sampling_node_list` or `merged_node_list`.

File: dingo/pipe/nodes/sampling_node.py [hunk 2]
```diff
@@ -65,3 +65,7 @@ def samples_file(self):
         return os.path.join(
             self.inputs.result_directory, self.label + ".hdf5"
         )
+
+    @property
+    def result_file(self):
+        return self.samples_file
```
Comment (mpuerrer) [inline dingo/pipe/nodes/sampling_node.py:71]: Given that `self.samples_file` is used should the TODO in line 64 be removed?

**Review by mpuerrer — COMMENTED** [review]
I'm not very familiar with the pipe aspect of the code, but AFAIK it looks sane! Perhaps also mention that summarypages is currently a stub.

File: dingo/pipe/nodes/sampling_node.py (hunk 2 above)
Comment (stephengreen) [reply dingo/pipe/nodes/sampling_node.py:71]: Ah, I have to check whether that's referred to. We need `result_file` for `PESummary`. But I'm not sure whether the other one is needed. I'll fix later.

File: dingo/pipe/dag_creator.py (hunk 1 above)
Comment (stephengreen) [reply dingo/pipe/dag_creator.py:142]: I agree the naming is brutal; I will fix in a later update since I do not want to risk making an error.

**Pull request merged**